Yoast/Yoast-SEO-for-TYPO3
# [TASK] Replaced javascript React classes with functional components, updated npm packages and fixed bug with multiple related keyphrases

## Summary

This PR can be summarized in the following changelog entry:

* Updated npm packages
* Replaced React classes with functional components
* Added extra analyze call if there are multiple related keyphrases due to bug in "yoastseo" package

## Relevant technical choices:

* The bug within "yoastseo" causes the analysis to not return the general analysis when there are multiple keyphrases (only analysis from the keyphrases are returned). Analysis is now called twice, once general and once for the keyphrases and results are being merged

## Test instructions

This PR can be tested by following these steps:

* Pull branch in v9, v10 and v11 and check if everything still works

## Quality assurance


Fixes #
